pkg/pillar/cmd/diag/diag.go

```diff
@@ -1052,26 +1052,8 @@ func myGet(ctx *diagContext, reqURL string, ifname string,
 	rv, err := zedcloud.SendOnIntf(context.Background(), zedcloudCtx, reqURL, ifname,
 		0, nil, allowProxy, ctx.usingOnboardCert, withNetTracing, false)
 	if err != nil {
-		switch rv.Status {
-		case types.SenderStatusUpgrade:
-			fmt.Fprintf(outfile, "ERROR: %s: get %s Controller upgrade in progress\n",
-				ifname, reqURL)
-		case types.SenderStatusRefused:
-			fmt.Fprintf(outfile, "ERROR: %s: get %s Controller returned ECONNREFUSED\n",
-				ifname, reqURL)
-		case types.SenderStatusCertInvalid:
-			fmt.Fprintf(outfile, "ERROR: %s: get %s Controller certificate invalid time\n",
-				ifname, reqURL)
-		case types.SenderStatusCertMiss:
-			fmt.Fprintf(outfile, "ERROR: %s: get %s Controller certificate miss\n",
-				ifname, reqURL)
-		case types.SenderStatusNotFound:
-			fmt.Fprintf(outfile, "ERROR: %s: get %s Did controller delete the device?\n",
-				ifname, reqURL)
-		default:
-			fmt.Fprintf(outfile, "ERROR: %s: get %s failed: %s\n",
-				ifname, reqURL, err)
-		}
+		fmt.Fprintf(outfile, "ERROR: %s, %s: %s, failed: %v",
+			ifname, reqURL, rv.Status.String(), err)
```

Comment:
Makes sense to use the new String here to simplify.

Reply:
Commit is removed.